keyboards/sdrakbs/sdrakb00/readme.md

```diff
@@ -0,0 +1,56 @@
+# sdrakb00
+
+![sdrakb00](https://raw.githubusercontent.com/drabaioli/SdraKb00/refs/heads/main/img/assembled_leds_on.jpg)
```

Comment:
This image needs to be rehosted on Imgur.

Reply:
I can do that, no problem. However when inspecting other keyboard's readmee with links to Imgur most of those links where not valid (expired?). I'd argue that github is a much more reliable source for this. Is it ok to keep the image on github (which would also take into account future changes to the image) or is it mandatory to use Imgur?

Reply:
An image hosted on a completely different repo is no more reliable than imgur, and you really should not be using GitHub as an image hosting service. We also cannot allow images in this repo; it's already a fairly hefty download as is.

Reply:
Ok, changed to use Imgur link.

As a side note, I randomly opened 20 Imgur links. More than half of the links are broken.

Reply:
Some of them are still valid, but Imgur seems to have changed how they handle the thumbnail suffix so that you cannot directly navigate to them (but they appear to still work when the readme is rendered). Add an `h` to the end of the filename and you'll see what I mean.